### ISSUE TITLE
Experiment neural information retrieval models

### ISSUE THREAD

**Issue by bwanglzu** [NeuIR/Linnaeus#1]
Hi @thiziri I noticed that you're deeply involved in MatchZoo, actually when I was reading the code in MatchZoo, there is a huge number of bugs and dirty code exist, and almost impossible to refactoring. So I started this project for experiment NeuIR models. If you're interested, please feel free to contribute.. I'm still working on it for 1-2 hours per day :)

**Comment by thiziri** [comment]
Hi @bwanglzu , Yes, I'm using MatchZoo a lot, actually I'm working with it all the day, I try to run its models on [TREC]([https://trec.nist.gov/]) datasets, so I try to adapt the MatchZoo working process in order to run on TREC datasets and enable evaluation with the [trec_eval](https://trec.nist.gov/trec_eval/) tool. I'm open to contributions, and all questions and suggestions are welcome.

**Comment by bwanglzu** [comment]
@thiziri Good to know that! Actually, I used to be spent a lot of time on multimedia information retrieval (search result diversification) and published several papers to ACMMM. I'm interested in text retrieval a lot, so this is the reason I started this one. I'm also adapting on TREC dataset, more concretely: [here](http://www.bigdatalab.ac.cn/benchmark/bm/Domain?domain=NeuIR). Guess I can finish CDSSM by this week, and finish up all the unit tests.

Good to know you're open to contributing! I'm motivated to make Linnaeus to the best framework for neural information retrieval.
